xonsh/prompt/gitstatus.py

```diff
@@ -93,6 +97,29 @@ def _get_tag_or_hash():
     return tag_or_hash if have_tag_name else _get_def("HASH") + hash_
 
 
+def _get_files_changed():
+    try:
+        changed = _check_output(["git", "diff", "--numstat"])
+    except subprocess.CalledProcessError as e:
+        return
```

Comment:
```suggestion
    except subprocess.CalledProcessError:
        return
```

`flake8` is unhappy about the unused `e` variable

Reply:
How do you run all the tests locally?   Usually I have tox.ini for all my python libs/projects and can simply run tox -e<env>

Reply:
I found the run-tests.xsh script and used that to verify flake8 locally